certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW

```diff
@@ -1283,6 +1323,15 @@ PROCEDURE ERRORHANDLER
  NEWLINE
  PRINT INDENT(1)+Q+ERRORMSG+Q+": "+Q+ERRORDETAIL(ERRORDETAILOFFSET)+Q
 
+ IF ERRORCODE=503 THEN
+  DO
+   PRINT ","
+   NEWLINE
+   CALL PRINTCURRENTSTATE
+   NEWLINE
+   PRINT INDENT(2)+ "}"
```

Comment:
Can we move the closing bracket inside the PRINTCURRENTSTATE procedure?  It is closing the currentState object.  The you can also remove the following from RETURNJSON1:
```
 PRINT INDENT(3)+"},"
 NEWLINE
```

Reply:
requested changes completed